src/core/Toast/Toast.tsx

```diff
@@ -131,22 +137,36 @@ export const Toast = (props: ToastProps) => {
       setHeight(height);
     }
   };
+  const animationRef = (ref: HTMLDivElement) => {
+    if (ref && animateOutTo) {
+      const { x, y } = ref.getBoundingClientRect();
+      setAnimateOutX(animateOutTo.getBoundingClientRect().x - x);
```

Comment:
Maybe save `getBoundingClientRect` result or destruct x and y to have one call for it.

Reply:
Added anchorRect variable to save animateOutTo.getBoundingClientRect() results